Python/Distance_Calculator_App/main.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+from geopy.geocoders import Nominatim
+from geopy import distance
+""" A program to calculate distance between two geo-locations
+    Parameters:
+        firstlocation (str): The starting location of user
+        secondlocation (str): The final location of user
+    
+"""
+def location():
+    geolocator = Nominatim(user_agent="geoapiExercises")
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--firstlocation', type=str, required=True)
+    parser.add_argument('--secondlocation', type=str, required=True)
+    args = parser.parse_args()
+
+    """ calculating longitude and latitude of entered locations """
```

Comment:
Rather than a string block, this is more of a comment. Make this a comment please